### PR TITLE
feat(cgimap): config changeset timeout and max payload

### PR DESCRIPTION
**Pull request by galta95**

### CODE REVIEW

File: helm/microcosm/templates/cgimap-deployment.yaml [hunk 1]
```diff
@@ -22,18 +22,21 @@ spec:
...
           {{- end }}
-          # ports:
-          #   - name: fcgi
-          #     containerPort: 8000
-          #     protocol: TCP
+          command:
+            - "/app/start.sh"
+          args:
+            - "--port=8000"
+            - "--instance=30"
```
Comment (CptSchnitz) [inline helm/microcosm/templates/cgimap-deployment.yaml]: Make it so this value can also be changed.

File: helm/microcosm/values.yaml [hunk 2]
```diff
@@ -155,6 +155,10 @@ ingestion:
         limits:
           memory: '2Gi'
           cpu: '2'
+      args:
+        CHANGESET_TIMEOUT_OPEN: 1 day
+        CHANGESET_TIMEOUT_IDLE: 1 day
+        MAX_PAYLOAD: "5000000"
```
Comment (CptSchnitz) [inline helm/microcosm/values.yaml]: This should also be copied to the `myvalues.example.yaml` file.

**Review by CptSchnitz — CHANGES_REQUESTED** [review]

File: helm/microcosm/templates/cgimap-deployment.yaml (hunk 1 above)
Comment (galta95) [reply helm/microcosm/templates/cgimap-deployment.yaml]: Done

File: helm/microcosm/values.yaml (hunk 2 above)
Comment (galta95) [reply helm/microcosm/values.yaml]: Done

File: helm/microcosm/templates/cgimap-deployment.yaml [hunk 3]
```diff
@@ -32,8 +32,8 @@ spec:
           command:
             - "/app/start.sh"
           args:
-            - "--port=8000"
-            - "--instance=30"
+            - --port={{ .Values.ingestion.pods.cgimap.args.PORT }}
```
Comment (CptSchnitz) [inline helm/microcosm/templates/cgimap-deployment.yaml]: restore the port back to its original value, there is not point changing at, and even if you do, you also need to change it in nginx.

**Review by CptSchnitz — CHANGES_REQUESTED** [review]

File: helm/microcosm/templates/cgimap-deployment.yaml (hunk 3 above)
Comment (galta95) [reply helm/microcosm/templates/cgimap-deployment.yaml]: are you serious? 
image

Comment (CptSchnitz) [reply helm/microcosm/templates/cgimap-deployment.yaml]: pretty sure i marked only line 36, if not my bad.

Comment (CptSchnitz) [reply helm/microcosm/templates/cgimap-deployment.yaml]: also this is why I wrote value and not values.

Comment (galta95) [reply helm/microcosm/templates/cgimap-deployment.yaml]: oh

Comment (galta95) [reply helm/microcosm/templates/cgimap-deployment.yaml]: Done

**Review by CptSchnitz — APPROVED** [review]

**Pull request merged**